### ISSUE TITLE
Quotation marks are rendered as &quot in 1.4.26

### ISSUE THREAD

**Issue by bbhtt** [vector-im/element-android#6445]
### Steps to reproduce

1. Reply to a message having a word with quotation marks
2. See that it's mangled

Here are the cases: 

![image](https://user-images.githubusercontent.com/62639087/177027773-44c3a46c-af3b-44cf-ac16-0def7551647a.png)

The event source for example:

```
{
  "content": {
    "body": "> <@...> test\n\n\"test\"",
    "format": "org.matrix.custom.html",
    "formatted_body": "<mx-reply><blockquote><a href=\"https://matrix.to/#/!MJOXUSmcmyjOXEjsbp:.../$0VkvhRhLoDGyfXa86lX704YSw2DaOOSYA0jdUq7SdQo?via=...\">In reply to</a> <a href=\"https://matrix.to/#/@...\">@...</a><br />test</blockquote></mx-reply>&quot;test&quot;",
    "msgtype": "m.text"
  },
  "room_id": "!MJOXUSmcmyjOXEjsbp:...",
  "type": "m.room.message"
}
```

#### What did you expect?

Rendered properly as `"`

#### What happened instead?

Rendered as `&quot`

### Operating system version

Andorid 10

### Application version and app store

1.4.26 Google PLay

### Will you send logs?

No

### Are you willing to provide a PR?

No

**Comment by bbhtt** [comment]
Probably a regression from https://github.com/vector-im/element-android/pull/6357

**Comment by cloudrac3r** [comment]
Correct, this is a regression from 6537. I figured out how to fix it, expect a PR.

**Comment by ouchadam** [comment]
Fixed in v1.4.28

**Comment by dreaddr** [comment]
When is that release expected?
